website/templatetags/permission_tags.py

```diff
@@ -2,22 +2,57 @@
 
 register = template.Library()
 
+
 def can_edit(user, obj):
     try:
         if user == obj.user or user.id == obj.uid:
-           return True
+            return True
         else:
-           return False
-    except:
+            return False
+    except BaseException:
         return False
+
+
 register.filter(can_edit)
 
+
 def is_moderator(user):
     try:
         if user.groups.count() > 0:
             return True
         else:
             return False
-    except:
+    except BaseException:
         return False
+
+
 register.filter(is_moderator)
+
+
+def comment_id(answer):
```

Comment:
This is not clear.
Can u explain?

Reply:
In get-question.html
line no. 355
{% render_field form.body class+="form-control body" id=answer|comment_id %}
comment_id function is being used to give an id to the CKEditor for posting the comment.
because id=comment{{ answer.id }} is not working.